src/components/Button/LinkButton.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import { ReactNode } from 'react';
+
+interface ButtonLinkProps {
+  href: string;
+  className?: string;
+  children?: ReactNode;
+  disabled?: boolean;
+  onClick?: () => void;
+  onSubmit?: () => void;
+  type?: 'button' | 'submit';
+}
+
+function getClassName(
+  className: string | undefined,
+  disabled: boolean
+): string {
+  const classMap: Record<string, string> = {
+    button_large: disabled ? 'button_large_active' : 'button_large_disActive',
+    button_medium: disabled
+      ? 'button_medium_active'
+      : 'button_medium_disActive',
+    button_small: disabled ? 'button_small_active' : 'button_small_disActive',
+    button_large_fill: disabled
+      ? 'button_large_active flex-1'
+      : 'button_large_disActive flex-1',
+  };
+
+  return classMap[className ?? ''] ?? className ?? '';
+}
```

Comment:
여기도 마찬가지로 아래처럼 바꿀수 있을거같네요

```suggestion
function getClassName(className: string | undefined, disabled: boolean): string {
  const classMap: Record<string, string> = {
    button_large: 'button_large_active button_large_disActive',
    button_medium: 'button_medium_active button_medium_disActive',
    button_small: 'button_small_active button_small_disActive',
    button_large_fill: 'button_large_active flex-1 button_large_disActive flex-1',
  };

  return disabled 
    ? classMap[className]?.split(' ')[0] 
    : classMap[className]?.split(' ')[1] 
    || className;
}
```

Reply:
여기도 마찬가지의 문제로...